### PR TITLE
Accept additional data to be logged as values

### PR DESCRIPTION
**Pull request by matee911**

### CODE REVIEW

File: falcon_telegraf/utils.py [hunk 1]
```diff
@@ -10,4 +10,16 @@ def merge_and_normalize_tags(*dicts: Dict[Any, Any]):
                     result[str(k)] = str(v)
                 except:
                     pass  # don't care
+    return result
+
+
+def merge_values(*dicts: Dict[Any, Any]):
```
Comment (NicholasSimons) [inline falcon_telegraf/utils.py]: Isn't this the same as merge_and_normalize_tags? Can they both be replaced by merge_something()

File: README.md [hunk 2]
```diff
@@ -32,12 +32,12 @@ Measurement names can be set when creating middleware, prefixed or autogenerated
...
 Creates a metric named hits-path/to/endpoint with one fields called `hits` with value 1.
-Executes *after* `on-*` method was called. Because of this you can add custom tags.
+Executes *after* `on-*` method was called. Because of this you can add custom tags and values.
 Example:
 ```python
 def on_get(self, request, response, some_path_variable):
-    request.context['my_new_tag'] = 'this_tags_value'
-    request.context['some_path_variable'] = some_path_variable  ## this is unnecessary - it's logged by middleware
+    request.context['telegraf_tags']['my_new_tag'] = 'this_tags_value'
+    request.context['telegraf_values']['some_path_variable'] = some_path_variable  ## this is unnecessary - it's logged by middleware
```
Comment (NicholasSimons) [inline README.md]: Are all values logged by middleware, i.e. is the comment still valid?

File: falcon_telegraf/utils.py (hunk 1 above)
Comment (NicholasSimons) [reply falcon_telegraf/utils.py]: e.g. metric type (tag/value) passed as parameter and if clause added to determine whether to cast as string

**Review by NicholasSimons — APPROVED** [review]
+1

File: README.md [hunk 3]
```diff
@@ -32,12 +32,12 @@ Measurement names can be set when creating middleware, prefixed or autogenerated
...
 Creates a metric named hits-path/to/endpoint with one fields called `hits` with value 1.
-Executes *after* `on-*` method was called. Because of this you can add custom tags.
+Executes *after* `on-*` method was called. Because of this you can add custom tags and values.
 Example:
 ```python
 def on_get(self, request, response, some_path_variable):
-    request.context['my_new_tag'] = 'this_tags_value'
-    request.context['some_path_variable'] = some_path_variable  ## this is unnecessary - it's logged by middleware
+    request.context['telegraf_tags']['my_new_tag'] = 'this_tags_value'
+    request.context['telegraf_values']['some_path_variable'] = some_path_variable
```
Comment (ekamil) [inline README.md:40]: Please highlight this change in a separate section of readme, like changelog

**Pull request merged**